fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java

```diff
@@ -124,61 +123,48 @@ private String getUsername(Authentication authentication) {
     return username;
   }
 
-  private boolean isAuthorized(String username,
```

Comment:
Appeared that this was not being used anywhere (unless some service is calling this via groovy?)

Reply:
these are the spel permission functions in the `@Pre/PostAuthorize` annotations and I'm not sure they get detected as used. This one might not be used but might need some greppy type searches to confirm

Reply:
cfieber@ is right that some of these methods will show up as unused, but are actually used in SPEL expressions. I don't think this one is though, based on this Github Org-level search: https://github.com/search?q=org%3Aspinnaker+isAuthorized&type=Code